packages/meteor-accounts-saml/saml_server.js

```diff
@@ -102,8 +102,9 @@ Accounts.registerLoginHandler(function(loginRequest) {
 	}
 
 	if (loginResult && loginResult.profile && loginResult.profile.email) {
+		const emailRegex = new RegExp(`^${ loginResult.profile.email }$`, 'i');
```

Comment:
Hey, you should escape the string to insert in regex, you can use `s.escapeRegExp` or `RegExp.escape`

Reply:
done